edudoc/staff-crash-course/staff_crash_course.md

```diff
@@ -46,20 +46,21 @@ Lastly, for team/committee internal discussion you can take a shortcut by removi
 
 ## Representation
 
-Every WCA Staff member will also have received login credentials for their own personal *@worldcubeassociation.org* email account hosted by GSuite.\
+Every WCA Staff member will also have received login credentials for their personal *@worldcubeassociation.org* email account hosted by GSuite.\
+
 This account must always be used when sending WCA-related emails. Exceptions may be made only if the Staff Member has issues with the GSuite Software in the region where they live.
 
 Configure your new GSuite account to include a signature that comprises your name and your position (and nothing else, like 3x3 PBs or YouTube channels!). Be aware that this signature will be visible to **any recipient, along with a legal disclaimer for recipients outside of the** *@worldcubeassociation.org* domain.
 
-Think twice before sending any form of private messages through this account. You should never use this email for any work that is not related to the WCA. Please refer to the [WCA Email Account Policy](wcadoc{documents/policies/internal/Email%20Account.pdf}) for details.
+Think twice before sending any form of private messages through this account. You should never use this email for any work that is unrelated to the WCA. Please refer to the [WCA Email Account Policy](wcadoc{documents/policies/internal/Email%20Account.pdf}) for details.
 
 ## Mailing Lists
 
 There are several mailing lists which all have their dedicated audience and purposes. All team and committee addresses can be found on the [contact page of the WCA](wca{contact}), along with a brief description of their scope of work.\
 Other significant mailing lists include:
 
   - [reports@worldcubeassociation.org](mailto:reports@worldcubeassociation.org) - Recipient channel for all Delegate reports, worldwide. You can opt-in on your WCA profile settings page.
-  - [staff@worldcubeassociation.org](mailto:staff@worldcubeassociation.org) - This is the channel where communication related to the entire organization takes place. Only the really heavy stuff like global discussions belong here.
+ - [staff@worldcubeassociation.org](mailto:staff@worldcubeassociation.org) - This is the channel where communications related to the entire organization takes place. Only the really heavy stuff like global discussions belong here.
```

Comment:
This should either be "communications [...] take place" or "communication [...] takes place"